### PR TITLE
Testing Support for LWC: Developer Docs

### PR DESCRIPTION
**Pull request by xyc**
### What does this PR do?
Developer documentation for testing support for LWC

[Netlify Deploy Preview](https://deploy-preview-1765--salesforcedx-vscode.netlify.com/tools/vscode/en/lwc/testing)

### What issues does this PR fix or reference?
Parent issue: #1703

@mmonacelli-ccx

### CODE REVIEW

File: docs/_articles/en/lwc/testing.md [hunk 1]
```diff
@@ -7,22 +7,34 @@ Install [`sfdx-lwc-jest`](https://developer.salesforce.com/docs/component-librar
 
 ## Explore Your LWC Tests
 
-The LWC Tests sidebar provides several features. Here, you can see all your LWC tests at a glance. You can run one test method, the test methods in one file, or all your tests. You can view the results of your last test run. And you can jump from those results to the corresponding lines in your code. To access this sidebar, click the beaker icon (hover text: Test) in the view bar on the left side of the VS Code window. (If you don’t see this icon, make sure that the project you have open in VS Code contains an sfdx-project.json file in its root directory.)
```
Comment (mmonacelli-ccx) [inline docs/_articles/en/lwc/testing.md]: Copy edit: LWC tests provide a central location for LWC tests and allow you to run tests on the code you write. You can test methods in one file, view the results of the last tests run, and navigate to the corresponding lines of code directly from the tests run. To access the test sidebar, click the beaker icon in the View bar. If you don't see this icon, make sure the project you have open in VS Code contains an sfdx-project.json file in its root directory.

File: docs/_articles/en/lwc/testing.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+## Explore Your LWC Tests
+
+The LWC Tests sidebar provides several features. Here, you can see all your LWC tests at a glance. You can run one test case, the test cases in one file, or all your tests. You can navigate to the corresponding lines in your test code. You can view the results of your last test run. To access this sidebar, click the beaker icon (hover text: Test) in the view bar on the left side of the VS Code window. (If you don’t see this icon, make sure that the project you have open in VS Code contains an sfdx-project.json file in its root directory.)
+
+To run selected tests, in the LWC Tests view, hover over the name of a test case or file to reveal a play icon. Click the play icon (hover text: SFDX: Run Lightning Web Component Test Case, SFDX: Run Lightning Web Component Test File) to run a test case or all the test cases in a file. To run all your tests, click the larger play icon at the top of the LWC Tests view (hover text: SFDX: Run All Lightning Web Component Tests).
+
+After you run tests, the blue icons next to your test files and test cases change to green icons (for passing tests), or red icons (for failing tests), or orange icons (for skipped tests).
```
[the remark this answers is not in the document]
Comment (xyc) [reply docs/_articles/en/lwc/testing.md]: orange *for* skipped tests?

**Comment by xyc** [comment]
Reviewed by @mmonacelli-ccx and integrated feedback on the docs.

File: docs/_articles/jp/lwc/testing.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Explore Your LWC Tests
+
+The LWC Tests sidebar provides a central location for you to run tests on the code you write. You can run one test case, the test cases in one file, or all your tests. You can also view the results of the last tests run and navigate to the corresponding lines of code directly from the tests. To access the test sidebar, click the beaker icon (hover text: Test) in the View bar. If you don't see this icon, make sure the project you have open in VS Code contains an sfdx-project.json file in its root directory.
+
+To run selected tests, hover over the name of a test case or file to reveal a play icon from the LWC test view. Click the play icon (hover text: SFDX: Run Lightning Web Component Test Case, SFDX: Run Lightning Web Component Test File) to run a test case or all the test cases in a file. To run all of tests in the view, click the larger play icon at the top of the LWC Tests view (hover text: SFDX: Run All Lightning Web Component Tests).
+
+Test results are noted as follows: green for passing tests, red for failing tests, or orange for skipped tests.
+
+To jump to the test file, or a test case, click the test name in the sidebar.
```
Comment (xyc) [inline docs/_articles/jp/lwc/testing.md]:
```suggestion
To jump to a test file, or a test case, click the test name in the sidebar.
```

File: docs/_articles/en/lwc/testing.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+## Explore Your LWC Tests
+
+The LWC Tests sidebar provides a central location for you to run tests on the code you write. You can run one test case, the test cases in one file, or all your tests. You can also view the results of the last tests run and navigate to the corresponding lines of code directly from the tests. To access the test sidebar, click the beaker icon (hover text: Test) in the View bar. If you don't see this icon, make sure the project you have open in VS Code contains an sfdx-project.json file in its root directory.
+
+To run selected tests, hover over the name of a test case or file to reveal a play icon from the LWC test view. Click the play icon (hover text: SFDX: Run Lightning Web Component Test Case, SFDX: Run Lightning Web Component Test File) to run a test case or all the test cases in a file. To run all of tests in the view, click the larger play icon at the top of the LWC Tests view (hover text: SFDX: Run All Lightning Web Component Tests).
+
+Test results are noted as follows: green for passing tests, red for failing tests, or orange for skipped tests.
+
+To jump to the test file, or a test case, click the test name in the sidebar.
```
Comment (xyc) [inline docs/_articles/en/lwc/testing.md]:
```suggestion
To jump to a test file, or a test case, click the test name in the sidebar.
```

**Comment by xyc** [comment]
@ntotten Megan (@mmonacelli-ccx) is the tech writer from Sales Cloud. She's worked with @blythesheldon and me to perform QA on this PR to ensure it aligns with doc standards. I've integrated feedback from her into this PR.

**Pull request merged**